cli/popper/commands/cmd_run.py

```diff
@@ -50,11 +50,18 @@
     is_flag=True,
 )
 @click.option(
+    '-e',
     '--engine',
-    help='Specify runtime for executing the workflow [default: docker].',
+    help='Specify runtime for executing the workflow.',
     type=click.Choice(['docker', 'singularity', 'vagrant']),
-    required=False,
-    default='docker'
+    required=False
```

Comment:
this can default to docker, and remove the `required=False`

Reply:
@ivotron if we default this to `docker` or the `--resource-manager` flag to `host`, then we would have no way to check if the flags were given or not, and hence we cant write the logic of prioritizing flags over config files.

Reply:
Oh, nevermind, based on what we discussed on slack over option preferences, this is how we should expose this option

Reply:
np, will be removing `required=False`

Reply:
> if we default this to `docker` or the `--resource-manager` flag to `host`, then we would have no way to check if the flags were given or not, and hence we cant write the logic of prioritizing flags over config files.

sorry, we were writing at the same time. You're right, I forgot what we had discussed on slack. Leaving `required=False` is what we should do in order to let the engine info be obtained from the config file

Reply:
sorry, no problem.